src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -187,6 +175,18 @@ public Builder saveLogsTo(String path) {
             return logCollector(FileLogCollector.fromPath(path));
         }
 
+        /**
+         * @deprecated Please use <code>shutdownStrategy(ShutdownStrategy.SKIP)</code> instead.
```

Comment:
do a `{@link Class#method}` to make it a link. and it'll auto do the code stuff too